pagopa/io-app
# fix: [IOAPPFD0-113] Removal of detrimental style from ListItem

## Short description
removed a {flex:1} from `IOListItemStyles.listItemInner`. this should have no effect on components that are already used in the Design System, while `ListItemInfo` used to break when in less closed environments.

This is the product of a back and forth with @dmnplb  while trying to use the `ListItemInfo` component in a new screen.

## List of changes proposed in this pull request
- removal of said style


## How to test
at the moment, the components that use this style are found in
- Design system
- ProfileMainScreen (ListItemInfoCopy, ListItemNav)

navigate to those pages and make sure everything is rendered correctly
